targets/tinyfpga_bx/base.py

```diff
@@ -0,0 +1,127 @@
+import sys
+import struct
+import os.path
+import argparse
+
+from migen import *
+from migen.genlib.resetsync import AsyncResetSynchronizer
+
+from litex.build.generic_platform import Pins, Subsignal, IOStandard
+from litex.soc.integration.soc_core import *
+from litex.soc.integration.builder import *
+
+from gateware import info
+from gateware import spi_flash
+
+from targets.utils import csr_map_update
+
+
```

Comment:
This seems duplicate with the `platform/tinyfpga_bx.py` file?

Reply:
I don't see `csr_map_update` imported anywhere in `platform/tinyfpga_bx/py`?

Reply:
I think this comment was actually attached to code you removed. Closing.